camayoc/tests/qcs/api/v1/conftest.py

```diff
@@ -0,0 +1,257 @@
+"""Pytest customizations and fixtures for the quipucords tests."""
+import os
+from pprint import pformat
+
+import pytest
+
+import requests
+
+from camayoc.config import get_config
+from camayoc.constants import (
+    VCENTER_CLUSTER,
+    VCENTER_DATA_CENTER,
+    VCENTER_HOST,
+)
+from camayoc.exceptions import WaitTimeError
+from camayoc.qcs_models import (
+    Credential,
+    Scan,
+    ScanJob,
+    Source,
+)
+from camayoc.tests.qcs.api.v1.utils import wait_until_state
+from camayoc.tests.utils import wait_until_live
+
+
+SCAN_DATA = {}
+"""Cache to associate the named scans with their results."""
+
+
+def run_scans():
+    """Check for run scans environment variable."""
+    result = True
+    run_scans = os.environ.get('RUN_SCANS', 'true')
+    if run_scans.lower() == 'false':
+        result = False
+    return result
```

Comment:
So is this something we would pass when we ran the tests?

i.e., after my docs PR gets merged we could run `make test-qpc RUN_SCANS=True` ?

Reply:
`RUN_SCANS` is True by default, so you don't have to pass it for the typical case its more if you wanted to run something like `make test-qpc-filter`, however, I don't see that one in your latest Makefile (https://github.com/quipucords/camayoc/blob/cdd12495fa93f357ece734f876aecaf43b797aec/Makefile).

So if you wanted to just run credential test then you would supply:
`RUN_SCANS=True make test-qpc-filter pattern=credential`

And not spend the 5+ minutes of the actual scans running.

Reply:
Yeah at @elyezer's suggestion I made it just so every target uses $(PYTEST_OPTIONS) so you can pass the `-k stringtofilterfor` option to any of the `test-qpc` or `test-rho` targets